heureka/ui/src/lib/slices/createFiltersSlice.js

```diff
@@ -3,15 +3,15 @@
  * SPDX-License-Identifier: Apache-2.0
  */
 
-import { produce } from "immer"
+import { produce, current } from "immer"
```

Comment:
whats current for - is it used?

Reply:
This helps for debugging, I remove it. It was leftover from debugging state content.